### PR TITLE
Make Lupus 12126 and 12127 OTA capable

### PR DESCRIPTION
**Pull request by cromefire**
There are OTA updates available and the devices should be and seem to be able to receive them (athough it seems minde is up-to-date, so I wasn't able to upload a new version)

### CODE REVIEW

**Comment by cromefire** [comment]
They also don't seem to be able to configure the power on behavior, but I don't know how to fix that:
image

File: src/devices/lupus.ts [hunk 1]
```diff
@@ -49,13 +49,15 @@ const definitions: Definition[] = [
         vendor: 'Lupus',
         description: '1 channel relay',
         extend: [onOff()],
```
Comment (Koenkk) [inline src/devices/lupus.ts:51]:
```suggestion
        extend: [onOff({powerOnBehavior: false})],
```

To fix the power on behaviour error

**Comment by cromefire** [comment]
Applied the changes.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**